py3status/i3status.py

```diff
@@ -143,6 +145,25 @@ def i3status_config_reader(self, i3status_config_path):
                 in_section = True
                 section_name = 'order'
 
+            if (not in_section) and line.startswith('group'):
```

Comment:
True, but sometimes it's nice to make things explicit

I'm happy to change this if there is a strong feeling